### PR TITLE
Added en homepage

### PR DESCRIPTION
**Pull request by windsonsea**
增加了英文首页

### CODE REVIEW

File: docs/index_en.md [hunk 1]
```diff
@@ -1,4 +1,87 @@
...
-template: overrides/home.html
-title: DaoCloud Enterprise
+hide:
+  - navigation
+  - toc
 ---
+
+# DaoCloud Enterprise 5.0
+
+On the occasion of the golden Autumn of 2022, DaoCloud gladly launches the next-generation, cloud-native, containerized, comprehensive platform by fusing the most popular cloud-native technologies in the open source community. After two years of continuous development, testing, and verification by hundreds of diligent developers, the flower of idealism finally blooms in the land of romantic open-source community. The new plaform leads the paces and waves of cloud-native development globally.
```
Comment (windsonsea) [inline docs/index_en.md:9]:
```suggestion
On the occasion of the golden Autumn of 2022, DaoCloud gladly launches the next-generation, cloud-native, containerized, comprehensive platform by fusing the most popular cloud-native technologies in the open source community. After two years of continuous development, testing, and verification by hundreds of diligent developers, the flower of idealism finally blooms in the land of romantic open-source community. The new platform leads the paces and waves of cloud-native development globally.
```

File: docs/index_en.md [hunk 2]
```diff
@@ -1,4 +1,87 @@
...
+
+Compatible with native Kubernetes and much more developed PaaS platform such as OpenShift, Tanzu, and more.
+This container manager is aimed to facilitate your cloud-native applications. You can centrally manage multiple clouds and clusters with this container manager,
+simplify your applications rolling out to any cloud, and significantly reduce your total cost of ownership. [Learn more](en/kpanda/03ProductBrief/WhatisKPanda.md)
+
+=== "Global Manager"
+
+Global manager is a user-oriented comprehensive service module, including user and access control, workspaces, audit logs, and personal settings. [Learn more](en/ghippo/01ProductBrief/WhatisGhippo.md)
+
+- User and access control: To securely manage access rights to resources, you can create, manage, delete users/groups, and flexibly configure user/group roles to complete the division of functional rights.
```
Comment (windsonsea) [inline docs/index_en.md:31]:
```suggestion
Global manager is a user-oriented comprehensive service module, including the user and access control, workspaces, audit logs, and personal settings. [Learn more](en/ghippo/01ProductBrief/WhatisGhippo.md)

- User and access control: To securely manage access rights to resources, you can create, manage, delete users/groups, and flexibly configure user/group roles to complete the division of functional rights.
```

**Review by windsonsea — COMMENTED** [review]
minor changes

File: docs/index_en.md [hunk 3]
```diff
@@ -1,4 +1,87 @@
...
+
+![img](zh/images/ops-tech.png)
+
+## Editions
+
+![modules](zh/images/dce-modules01.png)
+
+=== "Community Edition"
+
+    With the cloud native base as the core and many more self-developed plugins, this edition provides cloud native computing, network, storage, and other capabilities, support for multi-cloud and multi-cluster full lifecycle unified management, shield the complexity of the underlying infrastructure, and reduce the threshold for enterprises to use cloud native applications, improve application development efficiency.
```
Comment (Stella0621) [inline docs/index_en.md:71]: this edition provides cloud native computing, network, storage, multi-cloud and  multi-cluster full lifecycle unified management, shields...reduces....improves...

Comment (Stella0621) [reply docs/index_en.md:71]: 能否改成两句话

File: docs/index_en.md [hunk 4]
```diff
@@ -1,4 +1,87 @@
...
+
+    [Free Trial Now](en/dce/license0.md){ .md-button .md-button--primary }
+
+=== "Standard Edition"
+
+    This edition has a production-grade cloud-native base, compatible with a variety of underlying infrastructures, supporting for Xinchuang heterogeneity, cloud-edge collaboration, easy creation and management of large-scale multi-clusters, CI/CD pipelines for automated application delivery and deployment, innovative integration with GitOps, and progressive delivery. The app store includes many more apps with ten categories.
```
Comment (Stella0621) [inline docs/index_en.md:77]: ...multi-cluster. CI/CD  can realize automatic application delivery and deployment... The app store selects ecological applications in ten fields and provides a complete software stack for enterprise users.

Comment (Stella0621) [reply docs/index_en.md:77]: 同上和语法问题

File: docs/index_en.md [hunk 5]
```diff
@@ -1,4 +1,87 @@
...
+
+=== "Advanced Edition"
+
+    On the basis of the standard version, this edition adopts cutting-edge microservices and mesh technologies, supports for non-invasive access to traditional, cloud-native, and open source microservice frameworks, and uniformly realizes microservice lifecycle management, provides non-intrusive traffic governance features, to solve the service governance problem of users in the complex cloud-native environment of multi-cloud and multi-cluster.
```
Comment (Stella0621) [inline docs/index_en.md:81]: 同上，读着很奇怪

File: docs/index_en.md [hunk 6]
```diff
@@ -1,4 +1,87 @@
...
+    With the observability, this edition can trace through every traffic, automatically collects data, watch metrics of clusters, nodes, applications, and services in real time, and quickly locates faults.
+
+=== "Platinum Edition"
+
+    On the basis of the advanced version, this edition includes multi-cloud orchestration technology to realize centralized management of multi-cloud and hybrid clouds, provide cross-cloud application deployment, release, and O&M capabilities, support for scalability based on cluster resources, and achieve global load balancing. If any disaster occurs, this edition can effectively implement the failover.
```
Comment (Stella0621) [inline docs/index_en.md:86]: 同上

File: docs/index_en.md [hunk 7]
```diff
@@ -1,4 +1,87 @@
...
+    This edition includes many selected middleware such as databases, distributed messages and log retrieval, provides middleware management capabilities for the full lifecycle of multi-tenancy, deployment, observability, backup, and O&M, to keep your business with high availability.
```
Comment (Stella0621) [inline docs/index_en.md:87]: 同上

File: docs/index_en.md [hunk 8]
```diff
@@ -1,4 +1,87 @@
...
+- Workspace and folder: A resource isolation unit with hierarchical structure and access control, you can set the hierarchical structure according to the enterprise development environment and department structure.
+
+- Audit log: Provide you with logs of resource operation, through which you can quickly implement security analysis, resource changes, and troubleshooting.
+
+- Platform settings: This feature enable you custom the security policies, mails, and peronal appearance.
+
+=== "Observability"
+
+The observability module features with application-centric, out-of-the-box, cloud-native.
+It can monitor applications and resources in real time, collects data such as metrics, logs, and events to analyze application health status, and provides alerts, and comprehensive, clear, multi-dimensional data visualization capabilities. [Learn more](en/insight/03ProductBrief/WhatisKInsight.md)
```
Comment (Stella0621) [inline docs/index_en.md:42]: The application Insight is a new generation cloud native observability platform that focuses on applications analyzing. It provides the out-of-the-box real-time resources monitoring, metrics, logs, events and other data to help you analyzing  application status. In additionally, it is compatible with popular open source components, and provides alerting, multi-dimensions data visualization, fault locating, one-click monitoring and diagnosing capabilities.

Comment (Stella0621) [reply docs/index_en.md:42]: 可参考修改

**Review by Stella0621 — COMMENTED** [review]
一逗到底的句子建议拆开翻译

**Comment by windsonsea** [comment]
Thanks for your review.

rebase 出了点问题，我新开一个 PR 吧，汗

File: docs/index_en.md (hunk 6 above)
Comment (windsonsea) [reply docs/index_en.md:86]: 姐姐，review 要给出具体的修改建议....

File: docs/index_en.md (hunk 7 above)
Comment (windsonsea) [reply docs/index_en.md:87]: 同上    两个字无意义

**Pull request closed without merging**